projects/storefrontlib/src/lib/cms/facade/cms.service.spec.ts

```diff
@@ -19,7 +19,7 @@ const MockCmsModuleConfig: CmsModuleConfig = {
   }
 };
 
-const mockContentSlot: any[] = [
+const mockContentSlot: Component[] = [
```

Comment:
For this test, you can use type `Component` here, because the mocked data only has `uid` and `typeCode`.